api/specs.proto

```diff
@@ -84,7 +84,7 @@ message ServiceSpec {
 		// Aliases specifies a list of discoverable alternate names for the service on this Target.
 		repeated string aliases = 2;
 	}
-	repeated NetworkAttachmentConfig networks = 7;
```

Comment:
Yeah. I think I should be ablet to remove the definition of the embedded `NetworkAttachment` but keep the `networks` field there.